scwconfig/load.go

```diff
@@ -34,13 +36,15 @@ func Load() (Config, error) {
 		}
 		confV1, err := unmarshalConfV1(content)
 		if err == nil {
+			logger.Warningf("loaded config V1 from %s: config V1 is deprecated, please switch your config file to the V2", configPath)
```

Comment:
If the user see this he will be like OK but What should I do ? Isn't migration automatic ?

Reply:
It would be nice to add a link to a quick doc (e.g.: `https://github.com/scaleway/scaleway-sdk-go/blob/master/scwconfig/README.md`)